packages/cra-template-edu/template.json

```diff
@@ -0,0 +1,47 @@
+{
+  "package": {
```

Comment:
This file becomes the package.json of the generated app (mixed together with create-react-app's defaults).